### PR TITLE
[shelly] Add totalKWH channels for Shelly Pro 3EM

### PR DESCRIPTION
**Pull request by JonathanvdGHU**
# Added totalKWH channels for Shelly Pro 3EM
# Description
I added the missing `totalKWH` channels for the Shelly Pro 3EM, see #16772. There is also a new `totalKWH` channel created for the entire device. The channels for `reactivewatts` are left untouched because I couldn't locate the associated data.

After linking items to the `totalKWH` channels I noticed that the values only update when you restart OpenHAB at least once.
This also happens for other channels so it's not related to this pull request.

I am btw not really familiar with the code so there are probably some improvements.

### CODE REVIEW

**Comment by JonathanvdGHU** [comment]
@jlaur To be honest, it's a bug fix and an enchancement at the same time. The `totalKWH` channel for each meter already existed but some work was forgotten and that was why it didn't work.

File: bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly_de.properties [hunk 1]
```diff
@@ -300,6 +300,8 @@ channel-type.shelly.lastPower1.label = Schnitt letzte Minute
 channel-type.shelly.lastPower1.description = Stromverbrauch der letzten Minute, gerundet
 channel-type.shelly.meterTotal.label = Gesamtverbrauch
 channel-type.shelly.meterTotal.description = Gesamtverbrauch in kWh seit das Gerät eingeschaltet wurde (wird beim Neustart zurückgesetzt)
+channel-type.shelly.totalKWH.label = GesamtkWh (wird beim Neustart zurückgesetzt)
+channel-type.shelly.totalKWH.description = Gesamtverbrauchte Energie in kWh (wird beim Neustart zurückgesetzt)
```
Comment (lsiepel) [inline bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly_de.properties:304]: Please revert changes to this file. Translations are created in crowdin. 
For reference: https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations

**Review by lsiepel — CHANGES_REQUESTED** [review]
One question, otherwise LGTM

**Comment by JonathanvdGHU** [comment]
> One question, otherwise LGTM Channels do not seem to be created dynamically, so this would also require thing upgrade instructions.
> 
> For future PR's it would be nice to seperate the bug fix and the enhancement. This would make it possible to 1. easy backport bug fixes and 2. list the changes/fixes speerately in the release notes.

The `totalKWH` channel for each meter is created when data is written to it. So those are created dynamically. The `totalKWH` channel for the entire device also pops-up without changing anything other than replacing the current Shelly binding.

I don't mind to creating an extra pull request, so would you prefer I make a new one?

**Review by lsiepel — APPROVED** [review]

**Pull request merged**